README.md

```diff
@@ -39,7 +39,9 @@ The [master](https://github.com/pravega/spark-connectors) branch will always hav
 
 | Spark Version | Pravega Version | Java Version To Build Connector | Java Version To Run Connector | Git Branch                                                                        |
 |---------------|-----------------|---------------------------------|-------------------------------|-----------------------------------------------------------------------------------|
-| 3.0           | 0.9             | Java 11                         | Java 8 or 11                  | [master](https://github.com/pravega/spark-connectors)                             |
+| 3.0           | 1.0             | Java 11                         | Java 8 or 11                  | [master](https://github.com/pravega/spark-connectors)                             |
+| 2.4           | 1.0             | Java 11                         | Java 8                        | [r0.10-spark2.4](https://github.com/pravega/spark-connectors/tree/r0.10-spark2.4) |
```

Comment:
This `Java Version To Build Connector` should be `Java 8`.

FYI @claudiofahey , I will help to create this branch based on `r0.9-spark2.4` after this PR goes into master.

Reply:
Updated.